libs/data-types/src/lib/utilities/bootstrapDynamicTypes/bootstrapDynamicTypes.spec.ts

```diff
@@ -48,18 +50,21 @@ class Undecorated {
     bar: string;
 }
 
-const MachineUnion = (options: TypeDecoratorOptions) => Union(MACHINE_UNION, 'type', options);
+@Union(MACHINE_UNION, 'type')
+class MachineUnion {}
+
+const MachineUnionType = (options: TypeDecoratorOptions) => UnionType(MACHINE_UNION, options);
 
 class FactoryRoom {
-    @MachineUnion({
+    @MachineUnionType({
         label: 'machine',
         description: 'the machine that is in this factory room',
```

Comment:
We should be moving away from Detroit Fordist Manufacturing terminology and more towards land-based metaphors 😄